test/uxCapture.test.js

```diff
@@ -199,8 +202,17 @@ describe('UXCapture', () => {
 			).toBeTruthy();
 		});
 
+		it('should not thrown an exception if console.timeStamp is not available', () => {
```

Comment:
I don't think you really need a unit test for this case :)

If console.timeStamp starts throwing errors, it will do so in any of the other test cases that call `UXCapture.mark()`